### PR TITLE
Add screenshot build pipeline

### PR DESCRIPTION
**Pull request by ivy-lli**
- Update playwright
- Add screenshot project
- Add screenshot tests
- Build screenshot zip

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  apt-get install openjdk-17-jdk -y
+
+ARG MAVEN_VERSION=3.8.7
+ARG USER_HOME_DIR="/root"
+ARG SHA=21c2be0a180a326353e8f6d12289f74bc7cd53080305f05358936f3a1b6dd4d91203f4cc799e81761cf5c53c5bbe9dcc13bdb27ec8f57ecf21b2f9ceec3c8d27
+ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
+
+RUN mkdir -p /usr/share/maven /usr/share/maven/ref \
+  && curl -fsSL -o /tmp/apache-maven.tar.gz ${BASE_URL}/apache-maven-${MAVEN_VERSION}-bin.tar.gz \
+  && echo "${SHA}  /tmp/apache-maven.tar.gz" | sha512sum -c - \
```
Comment (ivy-rew) [inline build/Dockerfile:16]: looks hard to maintain (the SHA), do we really not trust these binaries and if they are corrupt, do we really care? :thinking:

File: integrations/standalone/tests/screenshots/project/.settings/ch.ivyteam.ivy.scripting.any [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  /imports {
+    "com.uipath.orchestrator.client.*"
```
Comment (ivy-rew) [inline integrations/standalone/tests/screenshots/project/.settings/ch.ivyteam.ivy.scripting.any:3]: stilll needed?

File: integrations/standalone/tests/screenshots/project/src_generated/com/twitter/status/gson/Entities_.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
+
+package com.twitter.status.gson;
+
+import javax.annotation.Generated;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+@Generated("org.jsonschema2pojo")
+public class Entities_ {
```
Comment (ivy-rew) [inline integrations/standalone/tests/screenshots/project/src_generated/com/twitter/status/gson/Entities_.java:9]: src_generated should not be put under version control

**Review by ivy-rew — APPROVED** [review]

File: build/Dockerfile (hunk 1 above)
Comment (ivy-lli) [reply build/Dockerfile:16]: Copy/paste from the glsp client tests, but I also don't like it.
I add a sub task to the story and try to improve that later

File: integrations/standalone/tests/screenshots/project/.settings/ch.ivyteam.ivy.scripting.any (hunk 2 above)
Comment (ivy-lli) [reply integrations/standalone/tests/screenshots/project/.settings/ch.ivyteam.ivy.scripting.any]: nope

**Pull request merged**